config/middlewares.js

```diff
@@ -1,7 +1,34 @@
+// all Devlaunchers internal projects domains
+const DevLaunchersSubdomains = [
+  'https://ideaspace.devlaunchers.org',
+  'https://ideaspace1.devlaunchers.org',
+  'https://ideaspace2.devlaunchers.org',
+  'https://projects.devlaunchers.org',
+  'https://projects1.devlaunchers.org',
+  'https://projects2.devlaunchers.org',
+  'https://recruitment.devlaunchers.org',
+  'https://recruitment1.devlaunchers.org',
+  'https://recruitment2.devlaunchers.org',
+  'https://website.devlaunchers.org',
+  'https://website1.devlaunchers.org',
+  'https://website2.devlaunchers.org',
+  'https://user-profile.devlaunchers.org',
+  'https://user-profile1.devlaunchers.org',
+  'https://user-profile2.devlaunchers.org',
+];
+
 module.exports = [
   'strapi::errors',
   'strapi::security',
-  'strapi::cors',
+  {
+    name: 'strapi::cors',
+    config: {
+      origin: ['https://devlaunchers.org', 'https://staging.devlaunchers.org', 'http://localhost:3000', ...DevLaunchersSubdomains],
```

Comment:
Nit: Technically https://staging.devlaunchers.org is also a subdomain.

Reply:
added while you're writing this :laughing:

Reply:
@chungthuang are you going to approve this now?